src/main/java/com/cloudbees/jenkins/plugins/bitbucket/server/events/NativeServerMirrorRepoSynchronizedEvent.java

```diff
@@ -31,8 +31,11 @@ public class NativeServerMirrorRepoSynchronizedEvent {
     private BitbucketServerMirrorServer mirrorServer;
 
     private BitbucketServerRepository repository;
+
     private List<NativeServerChange> changes;
 
+    private Boolean refLimitExceeded;
```

Comment:
why Boolean not a boolean  is this not required in the payload? (I see it in the datacenter 7.21 and 9.5 docs) (the whole event seems to be specific to BB Datacenter and not BB Cloud)?